### ISSUE TITLE
Problems with build

### ISSUE THREAD

**Issue by saurayi** [airgram/airgram#82]
**Issue type:**

[x] question
[ ] bug report
[ ] feature request
[ ] documentation issue

**Airgram version:**

[x] `latest`
[ ] `@next`
[ ] `2.3.x` (or put your version here)

Should I follow this step to build? https://github.com/tdlib/td#using-from-other-programming-languages

**Comment by esindger** [comment]
Hello, you should.
Airgram uses [td_json_client](https://core.telegram.org/tdlib/docs/td__json__client_8h.html), so you need to build TDLib according the [instruction](https://github.com/tdlib/td#building). Path to the library you can pass by the [command](https://airgram.io/api-reference/configuration#nodejs-version) option.

**Comment by eruslan44** [comment]
> Hello, you should.
> Airgram uses [td_json_client](https://core.telegram.org/tdlib/docs/td__json__client_8h.html), so you need to build TDLib according the [instruction](https://github.com/tdlib/td#building). Path to the library you can pass by the [command](https://airgram.io/api-reference/configuration#nodejs-version) option.

Hi! is it worth doing this if I use tdlib for Web?

**Comment by esindger** [comment]
No, you don't need to build TDLib if you want to install web version of the library:
```
npm install @airgram/web
```

**Comment by eruslan44** [comment]
> No, you don't need to build TDLib if you want to install web version of the library:
> 
> ```
> npm install @airgram/web
> ```

Thanks for the answer! 
I installed this package, ran the project, and got an error:
_http://localhost:8100/130f0ecf36ff90c87239.worker.js Not found._
My project uses angular, I can't figure out where I need to put the files from the tdweb package

**Comment by esindger** [comment]
I am not familiar with angular, but probably you need to copy worker files from `tdweb` package to the root directory of your project. See how it implemented in webpack example:

https://github.com/airgram/airgram/blob/768e87ad573b24fa2fe0c271d45bb1a6514c58dc/examples/webpack-config/webpack.config.js#L34-L42
